web3-functions/index.ts

```diff
@@ -0,0 +1,305 @@
+import {
+  Web3Function,
+  Web3FunctionContext,
+} from "@gelatonetwork/web3-functions-sdk";
+import { BigNumber, Contract } from "ethers";
+import { arrayify, toUtf8String, formatBytes32String } from "ethers/lib/utils";
+import { WrapperBuilder } from "@redstone-finance/evm-connector";
+
+const ORACLE_ABI = [
+  "function updateDataFeedsValuesPartial(bytes32[]) public",
+  "function getLastUpdateDetails(bytes32) public view returns (uint256, uint256, uint256)",
+  "function getLivePrice(bytes32[]) public view returns (uint256[], uint256)",
+];
+
+const redstone = require("redstone-protocol");
+
+type DataFeed = {
+  symbol: string;
+  id: string;
+  livePrice: BigNumber;
+  timestamp: number;
+  storedPrice: BigNumber;
+  storedTimestamp: number;
+};
+
+const isDebugMode = false;
```

Comment:
should this be mutable? or it's intended to be manually changed when debugging here?

Reply:
then all good. naming wise we could do
```
const DEBUG_MODE = false
```
but otherwise looks fine

Reply:
Done!